### ISSUE TITLE
Questions about training GAN

### ISSUE THREAD

**Issue by mitming** [POSTECH-CVLab/PyTorch-StudioGAN#79]
Hi @raven38 @Johnson-yue @MINGUKKANG , thanks for your contribution, I am wondering whether I can enlarge the batch size during training, but I am not sure if this is reasonable, could you give me some advice?

When training GAN, does increase the batch size also need to increase the learning rate linearly? Does the learning rate or batch size have a great impact on GAN training?

**Comment by wwjwy** [comment]
> Hi @raven38 @Johnson-yue @MINGUKKANG , thanks for your contribution, I am wondering whether I can enlarge the batch size during training, but I am not sure if this is reasonable, could you give me some advice?
> 
> When training GAN, does increase the batch size also need to increase the learning rate linearly? Does the learning rate or batch size have a great impact on GAN training?

https://arxiv.org/pdf/1706.02677.pdf   the reference said When the minibatch size is multiplied by k, multiply the learning rate by k

**Comment by MINGUKKANG** [comment]
Hi,

I am sorry for the late reply :(

In my experience, It was helpful to increase the learning rates (d_lr, g_lr) non-linearly as the batch size increased.

I used lr = \root(batch_size)*pre_defined_contant and experienced slight performance enhancement on CIFAR10 dataset.

Also, learning rate and batch size have a great impact on GAN training, but large values do not guarantee better results.

In the case of CIFAR10, 64 batch size gave the best result, but 128 batch size showed worse FID value.

Thank you.

**Comment by mitming** [comment]
Thanks a lot ! it is helpful !
